core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -349,6 +349,9 @@ public InternalLogId getLogId() {
 
     @Override
     public void run() {
+      if (lbHelper == null) {
```

Comment:
Let's add a comment like `// Workaround timer scheduled while in idle mode. This can happen from handleNotInUse() after an explicit enterIdleMode() by the user. Protecting here as other locations are a bit too subtle.`

And then it'll be clear it is load-bearing when we rework things to simplify.